developer/community.md

```diff
@@ -11,8 +11,8 @@ Reaction is a series of projects that combine the efforts of [community contribu
 
 ## Hop on a call with the team
 
--   **[Reaction Community Calls](http://getrxn.io/2rcCal):** Every other Wednesday at 9AM PST/12PM EST, join the core team for a public Reaction Community Call on YouTube Live to discuss the latest hot questions and topics from the community. Subscribe to our [Reaction Community Google Calendar](http://getrxn.io/2rcCal) to RSVP to the next call and check out the [agenda archives](https://docs.google.com/document/d/1PwenrammgQJpQfFoUUJZ96i_JJYCM_4glAjB1_ZzgwA/edit?usp=sharing) and [watch the archives](https://www.youtube.com/watch?v=wCctL5uCDNM&list=PLJ1TVRVOrm2N0G5zWfZms4Ef08m94k1VT).
--   **[Reaction Action](https://reactioncommerce.zoom.us/webinar/register/849ad115dc79936634538d7d4481ef37):** - RSVP for the monthly Reaction Action livestreams with guest speakers, video tutorials and more.
+-   **[Reaction Community Calls](http://getrxn.io/2rcCal):** Every other Wednesday at 7AM PST/12PM EST, join the core team for a public Reaction Community Call on YouTube Live to discuss the latest hot questions and topics from the community. Subscribe to our [Reaction Community Google Calendar](http://getrxn.io/2rcCal) to RSVP to the next call and check out the [agenda archives](https://docs.google.com/document/d/1PwenrammgQJpQfFoUUJZ96i_JJYCM_4glAjB1_ZzgwA/edit?usp=sharing) and [watch the archives](https://www.youtube.com/watch?v=wCctL5uCDNM&list=PLJ1TVRVOrm2N0G5zWfZms4Ef08m94k1VT).
```

Comment:
@machikoyasuda Not sure the archive link is the one you intended to be displayed. This one seems better: https://www.youtube.com/playlist?list=PLJ1TVRVOrm2N0G5zWfZms4Ef08m94k1VT